docs/translations/translations.json

```diff
@@ -85,6 +85,9 @@
   "toggleRTL": "Toggle right-to-left/left-to-right",
   "toggleTheme": "Toggle light/dark theme",
   "traffic": "Traffic",
+  "v5IsOut": "v5 beta is out! 🎉 Head to the",
```

Comment:
Feeling like having the emoji first works better:` 🎉 v5 beta is out! Head to the`